values.yaml

```diff
@@ -69,11 +69,13 @@ appContextRoot: /
 ## @param logback.globalLoggingLevel Global logging level. Possible values: "trace", "debug", "info", "warn", "error"
 ## @param logback.scanEnabled Enables scanning of logback.xml.
 ## @param logback.scanPeriod Interval for checking logback.xml configuration.
+## @param logback.fileLogging Enables file logging to log/pod_name.log
 ##
 logback:
   globalLoggingLevel: "info"
   scanEnabled: true
   scanPeriod: "30 seconds"
+  fileLogging: false
```

Comment:
It would be nice that we need any extra configuration for fileLogging, just documentation with steps how to configure it?